jaxrs-clients/src/main/java/com/palantir/remoting3/jaxrs/AbstractFeignJaxRsClientBuilder.java

```diff
@@ -91,8 +92,7 @@
                 .client(target.wrapClient(createOkHttpClient(userAgent)))
                 .retryer(target)
                 .options(createRequestOptions())
-                .logger(new Slf4jLogger(JaxRsClient.class))
-                .logLevel(Logger.Level.BASIC)
+                .logLevel(Logger.Level.NONE)
```

Comment:
wanted to make explicit that we're making a choice here. will add a comment to clarify.